tox.ini

```diff
@@ -29,6 +30,14 @@ deps =
     subprocess: subprocess32
 whitelist_externals = open
 
+[testenv:integration-tests]
+deps =
+    tox
+commands =
+    tox -e pypy-requests -- -k "tests/test_integration.py" {posargs:-vvs}
+    tox -e py27-requests -- -k "tests/test_integration.py" {posargs:-vvs}
+    tox -e py36-requests -- -k "tests/test_integration.py" {posargs:-vvs}
```

Comment:
running 3x serial runs of our heaviest test suite on the same shard may present a long pole in terms of the total CI runtime.. wonder if it would make sense to do one shard per interpreter + test_integration.py invocation here so that these can be parallelized? e.g. break out `pypy-integration`, `py27-integration`, `py36-integration` envs w/ backing shards.

wdyt?